patches/minecraft/net/minecraft/entity/LivingEntity.java.patch

```diff
@@ -170,30 +170,28 @@
  
 +      int i = net.minecraftforge.common.ForgeHooks.getLootingLevel(this, entity, p_213345_1_);
 +      this.captureDrops(new java.util.ArrayList<>());
-+
```

Comment:
But that needs toolchain work. Allowing automated remapping of mixins. Which I added to S2S.